plugins/adrv9009.c

```diff
@@ -1351,6 +1415,9 @@ static GtkWidget *adrv9009_init(struct osc_plugin *plugin, GtkWidget *notebook,
 			iio_toggle_button_init_from_builder(&subcomponents[i].rx_widgets[subcomponents[i].num_rx++],
 							subcomponents[i].iio_dev, subcomponents[i].ch1, "powerdown", builder,
 							"rx2_powerdown_en", 0);
+
```

Comment:
All signals connection are done a little bit latter. Look in source code for line: /* Connect signals */. Is there any reason this was needed here?

Reply:
the only reason to call it here is to tie it to label_sampling_frequency that corresponds to each device.